LICENSE

```diff
@@ -202,8 +202,8 @@
 
 ---------------
 
-pekko-sbt-paradox contains scripting adapted from paradox-material-theme version 0.6.0,
-which was released under an MIT license <https://github.com/jonas/paradox-material-theme>.
+pekko-sbt-paradox contains scripting adapted from sbt-paradox-material-theme version 0.7.0,
```

Comment:
I would prefer not to change the license - the original text is more accurate because that is where we borrowed the code from. Likewise for the links in the source code.

Reply:
You mean the license itself or the license file? The main reason for updating this is the repo location has changed and I am not sure how long I can guarantee github redirects.

Reply:
ok - since the repo has moved - then we can change the links

Can we change the version back though?

```
pekko-sbt-paradox contains scripting adapted from sbt-paradox-material-theme version 0.7.0
```

We adapted this code before 0.7.0 was released.

Reply:
@pjfanning Done, I checked and it lgtm but ill wait for your review just to make sure.